pom.xml

```diff
@@ -24,8 +24,8 @@
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
                     <encoding>UTF-8</encoding>
-                    <source>11</source>
-                    <target>11</target>
+                    <source>17</source>
```

Comment:
Since this is a lib, do we want to force the latest Java LTS version?

Reply:
In the upstreamed externalization code, there was one or more java17 switch expressions that I didn't want to backport to regular switch statements. Should not be a problem, though, if we decide that java11 shall be the baseline here.